src/main.rs

```diff
@@ -93,6 +100,14 @@ struct CliArgs {
     /// Show manifest size, XMP url and other stats.
     #[clap(long)]
     info: bool,
+
+    /// Name of an external process which will sign the claim bytes.
+    #[clap(long)]
+    signer_process: Option<String>,
```

Comment:
I was a little confused with the term "process" here at first,  wondering if it is expecting the name of a running os process or the path to an executable. I think the description should be something more like:
"Path to an executable that will sign the claim bytes"